scripts/dasharo-deploy

```diff
@@ -810,6 +810,12 @@ usage() {
   echo "  $0 restore  - Restore from a previously saved backup"
 }
 
+if check_if_intel; then
+  if ! can_install_dasharo; then
+    error_exit "Dasharo cannot be installed on this platform"
+  fi
+fi
```

Comment:
We should run it when Dasharo is not installed. We have a fused Boot Guard on our work laptops with Dasharo.

Reply:
https://github.com/Dasharo/dts-scripts/compare/d407c14bfb8efaccb8475b979738f1e46d6bffa6..4f681b5b519c67b505ffd19610bc8b6a4b2323cb